src/proof/PGInterCheck.C

```diff
@@ -700,8 +700,9 @@ bool ProofGraph::verifyGenSimultaneousAbstraction( vec< PTRef > & interps)
 	logic_.dumpHeaderToFile( dump_out );
 
 	// Add conjunction interpolants
-	for( unsigned i = 0; i < interps.size()-1; i++ )
-		logic_.dumpFormulaToFile( dump_out, interps[i], false );
+	for( int i = 0; i < interps.size()-1; i++ ) {
+        logic_.dumpFormulaToFile(dump_out, interps[i], false);
+    }
```

Comment:
Problem with indentation?

Reply:
The same problem as before.
Do you agree to reformatting the whole file to replace tab character with 4 spaces?

Reply:
Yes, as a general rule the code that I've touched I've tried to convert the indentation to 4 spaces.

Reply:
Fixed. Whole file now uses 4 spaces per indentation level.